tests/performance_tests/performance_test.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+
+from dataclasses import dataclass, asdict, field
+import os
+from platform import release, system
+import sys
+import json
+from datetime import datetime
+import shutil
```

Comment:
Can we group these between imports and froms?

Reply:
Done. https://github.com/AntelopeIO/leap/pull/302/commits/7e9117ec2f4e47755343bd6177e335f57ea9c32c